install-dev/init.php

```diff
@@ -123,10 +123,10 @@
 require_once _PS_INSTALL_PATH_.'classes/session.php';
 
 @set_time_limit(0);
-if (!@ini_get('date.timezone')) {
-    @date_default_timezone_set('Europe/Paris');
-    ini_set('date.timezone', 'UTC');
-}
+// Work around lack of validation for timezone
+// standards conformance, mandatory in PHP 7
+@date_default_timezone_set('Europe/Paris');
+ini_set('date.timezone', 'UTC');
```

Comment:
We could not force everyone to be in UTC.
Maybe try something like
```php
if (!in_array(@ini_get('date.timezone'), timezone_identifiers_list()) {
    @date_default_timezone_set('Europe/Paris');
    ini_set('date.timezone', 'UTC');
}
```

Reply:
Watch it, these two are twi different timezones.
```php
@date_default_timezone_set('Europe/Paris');
ini_set('date.timezone', 'UTC');
```

To be coherent, it should be:
```php
@date_default_timezone_set('Etc/UTC');
ini_set('date.timezone', 'UTC');
```

Reply:
@eternoendless and @PierreRambaud, your conclusion is correct and your suggestion is good. I'm reengineering the original hack, and testing the next set of changes before modifying the pull request.

Reply:
@PierreRambaud, your suggestion contained a fatal syntax error which I've corrected. The new code integrates @eternoendless's suggestion as well, and corrects all problems known in this pull request thread to date.

I think the pull request can now be merged, and request a code review to confirm this.